internal/gps/pkgtree/pkgtree_test.go

```diff
@@ -1280,6 +1280,11 @@ func TestListPackages(t *testing.T) {
 				},
 			},
 		},
+		"skip directories starting with '.'": {
+			fileRoot:   j("dotgodir"),
```

Comment:
Let's also add e.g. a `foo.go` dir to accompany the `.go` dir, so that we're testing both the dot-led and the dir-ending-in-.go case.